_clients/data-prepper/get-started.md

```diff
@@ -9,15 +9,42 @@ nav_order: 1
 
 Data Prepper is an independent component, not an OpenSearch plugin, that converts data for use with OpenSearch. It's not bundled with the all-in-one OpenSearch installation packages.
 
-## 1. Install Data Prepper
+If you are migrating from Open Distro Data Prepper, visit the [Migrating from Open Distro]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/migrate-open-distro/) page. 
 
-To use the Docker image, pull it like any other image:
+## 1. Installing Data Prepper
 
-```bash
+There are two ways to install Data Prepper for running:
+
+1. Run the Docker image
+2. Build from source
+
+The easiest way to use Data Prepper is by running the Docker image. We suggest
+you use this approach if you have [Docker](https://www.docker.com) available.
+
+You can pull the Docker image:
+
+```
 docker pull opensearchproject/data-prepper:latest
 ```
 
-## 2. Define a pipeline
+If you have special requirements that require you build from source, or if you
+want to contribute, please see the [Developer Guide](https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md).
+
+## 2. Configuring Data Prepper
+
+You must configure Data Prepper with a pipeline before running it.
+
+You will configure two files:
+
+* `data-prepper-config.yaml`
+* `pipelines.yaml`
+
+Depending on what you want to do, we have a few different guides to configuring Data Prepper.
+* [Trace Analytics]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/trace-analytics/) - Learn how to setup Data Prepper for trace observability
```

Comment:
@Naarcha-AWS , I still don't see these available in the documentation. I expect that these links will be broken.

Reply:
@dlvenable: I went ahead and made issue to migrate Trace Analytics before we merge this page. See issue #1837